### PR TITLE
v2.6.0

### PR DESCRIPTION
**Pull request by ht2**

### CODE REVIEW

**Comment by ht2** [comment]
I'm putting a hold on this right now. Things have got a bit out of whack and we're chasing our tail a bit.

**Comment by ryansmith94** [comment]
This pull request **introduces 6 alerts** and **fixes 3** when merging 343cce3c85eefac1318d90422c2fef1dd71214ff into 08ae4280620430f33e888f086ea3b46675f91798 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-acd85247fcafe3f9fd5543e1a7461122e62ef035)

**new alerts:**

* 5 for Superfluous trailing arguments
* 1 for Expression has no effect

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **fixes 3 alerts** when merging 0951fc89b28033d78ea19776501c93ecc35464b7 into 983e789520a79e9bded4822232ee22da2381c435 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-60b0eed4ba7685f518303d25b1197127dfa22284)

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **introduces 1 alert** and **fixes 3** when merging 8499ad41c3f9355030449535db5e2ba2b5fdf92c into 983e789520a79e9bded4822232ee22da2381c435 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-29157d87730326bad6cb45e4d63a670b600c99f1)

**new alerts:**

* 1 for Superfluous trailing arguments

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **introduces 1 alert** and **fixes 3** when merging 33ead4d55aa5ab7c608f26e01117c99f19b687f3 into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-26bc54a4cb5518b48719e5302032a94abbc593d9)

**new alerts:**

* 1 for Superfluous trailing arguments

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **fixes 3 alerts** when merging 7578a5140ab8a9e4935630ec5ca243388e30429e into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-8084e57b456b3908d17fa03a0c19e0c3a37397cb)

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **fixes 3 alerts** when merging 0f85fe5234f6dd9ea6c1e72dfb2657837b342cb4 into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-c859b3f7548870c9e4df93efd0a7386753b17997)

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **fixes 3 alerts** when merging 7b6fc19fc16ffaee39bedd0ba7ede9152cbff9f6 into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-55fb1d394a8201dd7aa2da94be5bb984a4c5f949)

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **introduces 1 alert** and **fixes 3** when merging 260a52f11983d3afffac0b3c7c143b7881f81328 into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-0452e6f84793cc86c8a3bf7e7a5d25579cf3ae5c)

**new alerts:**

* 1 for Superfluous trailing arguments

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **introduces 1 alert** and **fixes 3** when merging 7b95d8cf17731bf773ac9a6981c0f1178c2d3181 into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-909fb86659f5dab78fc526ca02b8beebecf8e1d4)

**new alerts:**

* 1 for Superfluous trailing arguments

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **fixes 3 alerts** when merging 52de07fe7bee03ee247f43983078ffcc57542380 into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-02b1a9cf615ce97924fbb1adc5a0b7e01d211272)

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: ui/src/components/Counter/index.js [hunk 1]
```diff
@@ -15,7 +15,17 @@ const formatLongNumber = x => x.toLocaleString();
 const formatTooltip = (count, benchmarkResult) => `Current: ${formatLongNumber(count)}   Previous: ${benchmarkResult}`;
 const getResultCount = rs => rs.getIn([0, 0, null, 'count'], 0);
 const getBenchmarkResultCount = rs => rs.getIn([1, 0, null, 'count'], 0);
-const makeHumanReadable = previewPeriod => previewPeriod.split('_').map(word => `${word.toLowerCase()} `).join('');
+const makeHumanReadable = (previewPeriod) => {
+  console.log('preview ', previewPeriod)
```
Comment (ht2) [inline ui/src/components/Counter/index.js]: Remove pls!

**Comment by ryansmith94** [comment]
This pull request **fixes 3 alerts** when merging 89bdd5d1cf40b13921d84299631ccd6490bd3cc3 into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-56872f9de011e141256a8ca8d45bcf74b4d0f037)

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ryansmith94** [comment]
This pull request **fixes 3 alerts** when merging 660020d864f116ad6aabfcaaaf0e87be84e130f4 into a6021af0fb33d97f2953c0ce876442cacaa4a299 - [view on LGTM.com](https://lgtm.com/projects/g/LearningLocker/learninglocker/rev/pr-383038f5f5a6a2266bc49ca0621b83c5cb67c0e3)

**fixed alerts:**

* 2 for Potentially inconsistent state update
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**